sidebars.js

```diff
@@ -21,24 +21,63 @@ const sidebars = {
       link: {
         type: 'generated-index',
       },
-      items: [
-        'installation/git',
-        'installation/docker',
-        'installation/helm',
-      ],
+      items: ['installation/git', 'installation/docker', 'installation/helm'],
     },
     {
       type: 'category',
       label: 'Log Ingestion',
       link: {
         type: 'generated-index',
+        description:
+          'Siglens default ingestion server runs on port: 8081. The below are the various ways to ingest logs into Siglens.',
       },
       items: [
-        'log-ingestion/fluentd-fluentbit',
-        'log-ingestion/vector',
-        'log-ingestion/splunk-hec',
+        {
+          type: 'category',
+          label: 'Vector',
+          link: {
+            type: 'generated-index',
+            title: 'Ingestion through Vector',
+            description: `Vector can be used to ingest logs from various sources. Please check the Vector documentation at https://vector.dev/docs/ for more information.`,
```

Comment:
In the netlify, this URL isn't showing up as a hyperlink for me: `https://vector.dev/docs/`.
Same comment for the URLs in the `description` parts of the Beats, Logstash, Fluentd categories below

Reply:
@AndrewHess Yes, I have tried to make them hyperlink. But these are written in the description and they are just string. So I am not able to make them Hyperlinks. 
But I still thought it would be good if this information were given on the initial home pages.

Let me know if you want me to remove the hyperlinks.